packages/front-end/services/track.ts

```diff
@@ -87,17 +87,28 @@ const dataWareHouseTrack = async (event: DataWarehouseTrackedEvent) => {
   if (!isTelemetryEnabled()) return;
 
   try {
-    await fetch(`${getIngestorHost()}/track?client_key=${GB_SDK_ID}`, {
-      method: "POST",
-      body: JSON.stringify(event),
-      headers: {
-        Accept: "application/json",
-        "Content-Type": "application/json",
-      },
-      // TODO: Make ingestor accept text/plain content type so we can disable cors
-      //credentials: "omit",
-      //mode: "no-cors",
-    });
+    const result = await fetch(
+      `${getIngestorHost()}/track?client_key=${GB_SDK_ID}`,
+      {
+        method: "POST",
+        body: JSON.stringify(event),
+        headers: {
+          Accept: "application/json",
+          "Content-Type": "text/plain",
+        },
+        credentials: "omit",
+      }
+    );
+    if (inTelemetryDebugMode()) {
+      if (!result.ok) {
+        console.error(
+          `Telemetry - Ingestor returned a ${result.status}`,
+          await result.json()
+        );
+      } else {
+        console.log("Telemetry - Successfully recorded event");
+      }
+    }
```

Comment:
I don't think we need to log successes.  And since this is wrapped in a try/catch already with logging, we can simply do:

```ts
const result = await fetch(...);
if (!result.ok) {
  throw new Error(...)
}
```

Reply:
done